yahoo-finance/YahooFinance.py

```diff
@@ -0,0 +1,12 @@
+from yahoo_finance import Share
+
+def get_data(ticker, start_date, end_date):
+	"""Creates wrapper instance of yahoo_finance Share class.
```

Comment:
Also, maybe we should make `end_date` default to today? Might be a nicer API. Something like:
```Python
import datetime as dt

def get_stock_prices(ticker, start_date, end_date=None):
    if end_data is None:
            end_data = dt.date.today()
    ...
```